test/src/main/java/org/apache/accumulo/test/functional/ManyWriteAheadLogsIT.java

```diff
@@ -87,11 +88,13 @@ public void alterConfig() throws Exception {
     try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
       InstanceOperations iops = client.instanceOperations();
       Map<String,String> conf = iops.getSystemConfiguration();
+      AccumuloConfiguration aconf = getCluster().getSiteConfiguration();
+      @SuppressWarnings("deprecation")
+      Property deprecatedProp = Property.TSERV_WALOG_MAX_SIZE;
       majcDelay = conf.get(Property.TSERV_MAJC_DELAY.getKey());
-      walogSize = conf.get(Property.TSERV_WALOG_MAX_SIZE.getKey());
-
+      walSize = conf.get(aconf.resolve(Property.TSERV_WAL_MAX_SIZE, deprecatedProp).getKey());
       iops.setProperty(Property.TSERV_MAJC_DELAY.getKey(), "1");
-      iops.setProperty(Property.TSERV_WALOG_MAX_SIZE.getKey(), "1M");
+      iops.setProperty(aconf.resolve(Property.TSERV_WAL_MAX_SIZE, deprecatedProp).getKey(), "1M");
```

Comment:
These can just set the new property, and don't need to make a decision based on what is set already in the config. First, we control the config for the test cluster, and you've already set the cluster to use the new property, and second, even if a cluster did use the old property in its existing config, setting only the new property is sufficient, since the new property will overrule the old one.